x/staking/types/msg.go

```diff
@@ -257,17 +257,17 @@ type MsgBeginRedelegate struct {
 	DelegatorAddress    sdk.AccAddress `json:"delegator_address"`
 	ValidatorSrcAddress sdk.ValAddress `json:"validator_src_address"`
 	ValidatorDstAddress sdk.ValAddress `json:"validator_dst_address"`
-	SharesAmount        sdk.Dec        `json:"shares_amount"`
+	Amount              sdk.Coin       `json:"amount"`
```

Comment:
`MsgDelegate` hasn't been updated, (i.e still says `Value` instead of `Amount`)

Reply:
https://github.com/cosmos/cosmos-sdk/blob/fbe8e896dd201ad26231a8ae33c42238af41f75b/x/staking/types/msg.go#L215

Reply:
good catch @fedekunze

Reply:
Indeed, updated :)